rfc/rfc019-employee-identity-means.md

```diff
@@ -0,0 +1,187 @@
+|                           | S. van der Vegt |
+|                           | Nedap           |
+|                           | April 2023      |
+
+## Employee Identity Authentication Means
+
+### Abstract
+
+The NutsEmployeeIdentity method is an authentication _means_ which allows an organisation to assert the identity of the current user.
+The organisation uses the information of the current user session to present the user with a statement about its identity and asks the user to confirm this.
+After confirmation this bundle of information is signed with the organisation's private key.
+
+This method makes it possible to assert a user identity without the need for a personal authentication means.
+This puts the organisation in the role of identity issuer instead of a trusted third party.
+A use-case can choose to allow this authentication under certain (low risk) circumstances which still require a person to be authenticated in order to comply with the applicable legislation.
+
+This RFC is an addition to the means listed in [RFC002](./rfc002-authentication-token.md#7-supported-means)
+
+### Status of document
+
+This document is currently in draft.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1.  Introduction
+
+Introducing new authentication means in an organizational setting can have quite an impact, even more when mainstream identity providers have not yet adjusted to new standards.
+New feature or applications require a higher assurance level for authentication than the current situation.
+This introduces an additional authentication step in the process of users. This often happens at a place in the workflow where they don't expect it.
+This is a situation that remain until the main login is adapted to the newer/higher security standard.
+This RFC bridges the gap so the software can already be changed to accept newer protocols and the user can be made aware of upcoming changes without going through an authentication step using additional means.
+
+## 2. Terminology
+
+* **Client application**: The application that requires access.
+* **Node**: A piece of software implementing the Nuts RFCs.
+* **Resource server**: The application \(a protected resource\) that requires authorized access to its API’s.
+* **VerifiableCredential**: Verifiable Credential according to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+* **VerifiablePresentation**: Verifiable Presentation according to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+## 3. Nuts Employee Identity
+
+The authentication flow is as follows:
+
+- a user requests access to a resource that is hosted at a *resource server* and requires additional authentication.
+- the *client application* starts a session on the *node* with the current user data.
```

Comment:
Maybe coin the term "user data" (maybe change it to "user session info/data") as "the user data from the existing authenticated session in the client application", or something?

Reply:
"user data from the active session"